static/js/controllers/user-language-modal.js

```diff
@@ -0,0 +1,64 @@
+  var inboxServices = angular.module('inboxServices');
+
+  inboxServices.factory('UserLanguageModal', ['$uibModal',
```

Comment:
Seems strange to register a factory in `/controllers/`. Can you pull this out in to `/services/`?